pkg/apis/crd/v1alpha1/types.go

```diff
@@ -600,3 +605,18 @@ type NamespacedName struct {
 	Name      string `json:"name,omitempty"`
 	Namespace string `json:"namespace,omitempty"`
 }
+
+// NetworkPolicyProtocol includes all protocols that `ports` can't support. All
```

Comment:
NetworkPolicyProtocol defines additional protocols that are not supported by "Ports".

Reply:
Done.